src/components/App.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './App.css';
+import { BrowserRouter, Route } from 'react-router-dom'
+import Navbar from "./Navbar"
+import CountriesList from "./CountriesList"
+import CountryDetails from "./CountryDetails"
+
+
+
+function App() {
+
+ 
+  return (
+    <main className="container-flex">
+      <Navbar />
+      <div className="row">
+
+        <BrowserRouter>
+          <Route path="/" render={() => <CountriesList />} />
+      
+        <Route path="/:cca3" render={props => <CountryDetails {...this.props} />} />
```

Comment:
```suggestion
        <Route path="/:cca3" render={props => <CountryDetails {...props} />} />
```
Esta que es la ruta en cuestión.

Tampoco te hace falta usar el _this_ porque estás apuntando a props en el scope del propiedad _render_